src/main/resources/swagger/wiremock-admin-api.json

```diff
@@ -1832,7 +1911,7 @@
         ]
       },
       "after-pattern": {
-        "title": "Before datetime",
+        "title": "After datetime",
```

Comment:
This change (and the rest of the changes in this file) are unrelated and the result of running `cd ui && npm run swagger` to regenerate this file. Probably should automate that as part of the build process.